sensing/autoware_pointcloud_preprocessor/include/autoware/pointcloud_preprocessor/distortion_corrector/distortion_corrector.hpp

```diff
@@ -47,74 +47,95 @@
 namespace autoware::pointcloud_preprocessor
 {
 
-class DistortionCorrectorBase
+struct AngleConversion
 {
-public:
-  virtual bool pointcloud_transform_exists() = 0;
-  virtual bool pointcloud_transform_needed() = 0;
-  virtual std::deque<geometry_msgs::msg::TwistStamped> get_twist_queue() = 0;
-  virtual std::deque<geometry_msgs::msg::Vector3Stamped> get_angular_velocity_queue() = 0;
-
-  virtual void processTwistMessage(
-    const geometry_msgs::msg::TwistWithCovarianceStamped::ConstSharedPtr twist_msg) = 0;
-  virtual void processIMUMessage(
-    const std::string & base_frame, const sensor_msgs::msg::Imu::ConstSharedPtr imu_msg) = 0;
-  virtual void setPointCloudTransform(
-    const std::string & base_frame, const std::string & lidar_frame) = 0;
-  virtual void initialize() = 0;
-  virtual void undistortPointCloud(bool use_imu, sensor_msgs::msg::PointCloud2 & pointcloud) = 0;
+  // Equation for the conversion between sensor azimuth coordinates and Cartesian coordinates:
+  // sensor azimuth coordinates = offset_rad + sign * cartesian coordinates;
+  // offset_rad is restricted to be a multiple of 90, and sign is restricted to be 1 or -1.
+  float offset_rad{0};
+  float sign{1};
+  static constexpr float offset_rad_threshold{0.087f};  // (5 / 180) * M_PI
```

Comment:
out of ignorance, can't you write `(5 / 180) * M_PI` in a constexpr?

Reply:
Thanks! fixed in https://github.com/autowarefoundation/autoware.universe/pull/8380/commits/bbbc501bfac443c533ecd1057e07f6d8487de8e0